<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintOptionsPage</class>
 <widget class="QWidget" name="PrintOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="mGroupImage">
     <property name="title">
      <string>Image Position</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="mPositionFrame">
           <zorder>mGroupScaling</zorder>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="kcfg_PrintAutoRotate">
           <property name="text">
            <string>Auto rotate</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="mPreview">
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="mLeftButton">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mRightButton">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>297</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGroupScaling">
     <property name="title">
      <string>Scaling</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="mNoScale">
        <property name="text">
         <string>&amp;No scaling</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleToPage">
        <property name="text">
         <string>&amp;Fit image to page</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_PrintEnlargeSmallerImages">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Enlarge smaller images</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>188</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleTo">
        <property name="text">
         <string>&amp;Scale to:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="kcfg_PrintWidth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string comment="Dimension separator, as in: '15 x 10 centimeters'">x</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="kcfg_PrintHeight">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_PrintUnit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Millimeters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centimeters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inches</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_PrintKeepRatio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Keep ratio</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>98</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mSaveSettings">
          <property name="text">
           <string>Save settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="mGroupPxP">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Photos per page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mPhotoXPage">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">kcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintUnit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintKeepRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleToPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintEnlargeSmallerImages</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mGroupScaling</sender>
   <signal>toggled(bool)</signal>
   <receiver>mSaveSettings</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mGroupPxP</sender>
   <signal>toggled(bool)</signal>
   <receiver>mGroupScaling</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mGroupPxP</sender>
   <signal>toggled(bool)</signal>
   <receiver>mGroupImage</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mGroupPxP</sender>
   <signal>toggled(bool)</signal>
   <receiver>mPhotoXPage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
